src/seedsigner/views/psbt_views.py

```diff
@@ -41,9 +41,6 @@ def run(self):
                 # Doesn't look like this seed can sign the current PSBT
                 button_str += " (?)"
             
-            if seed.passphrase is not None:
-                # TODO: Include lock icon on right side of button
-                pass
```

Comment:
Decided that it was not desired to have a lock icon as described in this comment (see [here](https://github.com/SeedSigner/seedsigner/pull/286#issuecomment-1363351268) for discussion). Removing comment and if statment.

Reply:
TY @EverydayBitcoiner,  this improvement has been borrowed (as an elif at same level) in pr #323

Reply:
TY @EverydayBitcoiner, this code cleanup has been included in pr #324